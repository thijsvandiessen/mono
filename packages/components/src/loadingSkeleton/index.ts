export { LoadingSkeleton } from './loadingSkeleton.jsx'
