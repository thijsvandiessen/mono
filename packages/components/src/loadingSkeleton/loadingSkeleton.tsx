import React from 'react'
import classNames from 'classnames'
import styles from './styles.module.scss'

export const LoadingSkeleton = () => (
  <div className={classNames(styles.loadingWrapper)} />
)
