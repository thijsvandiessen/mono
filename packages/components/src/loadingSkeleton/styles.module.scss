.loadingWrapper {
  height: 100vh;
}
