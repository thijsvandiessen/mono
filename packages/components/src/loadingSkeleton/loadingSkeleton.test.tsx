import React from 'react'
import { LoadingSkeleton } from './loadingSkeleton'
import { render } from '@testing-library/react'
import { describe, it, expect } from 'vitest'

describe('Loading skeleton component', () => {
  it('should render', () => {
    const { container } = render(<LoadingSkeleton />)
    expect(container).toMatchSnapshot()
  })
})
