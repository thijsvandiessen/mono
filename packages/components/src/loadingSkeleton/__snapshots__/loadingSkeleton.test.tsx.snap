// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loading skeleton component > should render 1`] = `
<div>
  <div
    class="_loadingWrapper_a0a619"
  />
</div>
`;
